lib/mri/rubygems.rb

```diff
@@ -1261,3 +1261,13 @@ def clear_default_specs
 require 'rubygems/core_ext/kernel_require'
 
 Gem.use_gemdeps
+
+# Add TruffleRuby rubygems hooks to install and uninstall executables from additional
+# GraalVM bin directories (./bin, ./jre/bin)
+if Truffle.graalvm?
+  require 'rubygems/installing_extra_executables'
```

Comment:
Maybe this should be `truffle/rubygems/installing_extra_executables` for clarity, although the filename seems unlikely to conflict.

Reply:
I don't think It would work our `./lib` is not on load-path.

Reply:
What I meant is moving `lib/truffle/rubygems/installing_extra_executables.rb` to `lib/truffle/truffle/rubygems/installing_extra_executables.rb`, there are other cases like that.

Reply:
ah, that would work, however this is not a truffle specific extension, it would work without truffle as well and since the collision risk is minimal, I'd rather keep it where it is.